witness/README.md

```diff
@@ -1,3 +1,116 @@
 # Disciplina witness
 
 Main node/witness library and executable
+
+## Building from source code
+
+```bash
+> stack build disciplina-witness
+```
+
+Dependencies:
+
+* [Stack](https://docs.haskellstack.org/en/stable/README/) tool
+* [RocksDB](https://github.com/facebook/rocksdb/blob/master/INSTALL.md)
+
+## Running witness node
+
+```bash
+> stack exec dscp-witness -- <parameters>
+```
+
+For ready scripts see [scripts section](#scripts).
+
+### Command-line arguments
+
+For the most actual information run
+
+```bash
+> stack build disciplina-witness -- --help
+```
+
+```
+Usage: dscp-witness [--version] [--debug] [--log-config FILEPATH]
```

Comment:
Let's not include this part. Help page will change and it's just not the most helpful information we can share with user.

Reply:
@flyingleafe wants to see it :man_shrugging:

Reply:
Yeah I still want to get rid of it.

Reply:
@flyingleafe Say something. I'd suggest you expected it to look differently, although I don't understand what can help it.